packages/react-native-renderer/src/legacy-events/SyntheticEvent.js

```diff
@@ -77,10 +76,8 @@ function SyntheticEvent(
   this.dispatchConfig = dispatchConfig;
   this._targetInst = targetInst;
   this.nativeEvent = nativeEvent;
-  if (!enableModernEventSystem) {
-    this._dispatchListeners = null;
-    this._dispatchInstances = null;
-  }
+  this._dispatchListeners = null;
```

Comment:
We don't need these. I think you got the condition the wrong way around here.

Reply:
These are in the native renderer. The native renderer uses the old event system, so we need these.